client/state/data-layer/wpcom/me/transactions/source-payment/index.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { SOURCE_PAYMENT_TRANSACTION_DETAIL_FETCH } from 'state/action-types';
+import fromApi from './from-api';
+import toApi from './to-api';
+
+export const fetchSourcePaymentTransactionDetail = action =>
+	http(
+		{
+			method: 'GET',
+			path: '/me/transactions/source-payment',
+			apiNamespace: 'rest/v1',
+			query: toApi( action ),
+		},
+		action
+	);
+
+export const onSuccess = () => {
+	return {};
+};
+
+export const onError = () => errorNotice( translate( 'Sorry. Something went wrong!' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Oops, something went wrong.' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/poc/add-pending-checkout-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).